applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -2461,7 +2460,10 @@ void SmallStrainUPwDiffOrderElement::
 {
     if (rVariables.UseHenckyStrain) {
         this->CalculateDeformationGradient(rVariables, GPoint);
-        this->CalculateHenckyStrain( rVariables );
+        const GeometryType& rGeom = GetGeometry();
+        const SizeType Dim        = rGeom.WorkingSpaceDimension();
```

Comment:
I would simplify this to:
```suggestion
        const SizeType Dim        = GetGeometry().WorkingSpaceDimension();
```

Reply:
Yes, done